server/graphql/v1/mutations/orders.js

```diff
@@ -231,6 +231,45 @@ const hasPaymentMethod = order => {
   }
 };
 
+export const deletePaymentMethod = async (collective, fingerprint, transaction) => {
+  // Check if the credit card is already saved
+  const oldPaymentMethod = await models.PaymentMethod.findOne(
+    {
+      where: {
+        CollectiveId: collective.id,
+        service: 'stripe',
+        saved: true,
+        data: {
+          fingerprint: fingerprint,
+        },
+      },
+    },
+    { transaction },
```

Comment:
As per https://sequelize.org/api/v6/class/src/model.js~model#static-method-findOne, the transaction is expected in 1st object. Typescript would have detected this.

Reply:
Done. 👍🏽